pages/nft/url/index.vue

```diff
@@ -240,9 +241,11 @@ export default class FetchIndex extends Vue {
 
   async onSubmit() {
     try {
+      logTrackerEvent(this, 'NFTUrlMint', 'onSubmit', this.state, 1);
       this.isLoading = true
       await this.doAction()
     } catch (err) {
+      logTrackerEvent(this, 'NFTUrlMint', 'onSubmitErr', (err as Error).toString(), 1);
```

Comment:
It's better not to log error here, since we do log errors in each function called by onSubmit
but we would want to log errors related to `case State.INIT:` including some expected ones (insufficient balance, etc)